docs/apidocs/qiskit.rst

```diff
@@ -7,20 +7,27 @@ API Reference
 .. toctree::
    :maxdepth: 1
```

Comment:
I think we'll want to increase this since we have more levels of hierarchy now. Like I was not able to get to the QuantumCircuit on the side bar with only 1 level deep now.

Reply:
This parameter does not change the sidebar expansion, just the main page TOC listing which would get a bit out of hand as you need to go to depth 3 now to get `QuantumCircuit`.